inspire/modules/formatter/views.py

```diff
@@ -84,3 +89,35 @@ def get_latex_file(recid, latex_format):
             )
         }
     )
+
+
+@blueprint.route("/export-as/<path:export_format>",
+                 methods=['POST'])
+@wash_arguments({'ids': (list, [])})
+def download_all_latex(ids, export_format):
```

Comment:
So where is the limit of records to download? I don't see where the 5000 record limit is set.